cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/CaseManager.cs

```diff
@@ -5335,7 +5424,8 @@ public async Task<SetCaseFlagsReply> SetCaseFlags(string dmerIdentifier, bool is
                             dfp_id = flag.Id,
                             dfp_description = flag.Description,
                             dfp_label = SanitizeLabel(flag.Description), // max 200 characters.
-                            dfp_type = (int?)FlagTypeOptionSet.Review
+                            dfp_type = (int?)FlagTypeOptionSet.Review,
+                            dfp_formid = flag.FormId
```

Comment:
note to self: SetCaseFlags is only used by UpdateCase which does not seem to be used by any project. Should be able to delete this change and mark this method "Obsolete". I've only seen it mentioned in commented out code